<launch>
  <node pkg="proto_ros" type="calib_vicon_capture_node" name="calib_vicon_capture_node" required="true" output="screen">
    <param name="rosbag_path" value="/data/vicon_capture.bag" />
    <param name="config_file" value="$(find proto_ros)/config/calib_vicon-intel_d435i.yaml" />
    <param name="cam0_topic" value="/camera/color/image_raw" />
    <!-- <param name="body0_topic" value="/vicon/f450/pose" /> -->
    <!-- <param name="target0_topic" value="/vicon/AprilGrid/pose" /> -->

    <!-- <param name="body0_topic" value="/vrpn_client_node/ucl_0/pose" /> -->
    <param name="body0_topic" value="/ucl_0/vrpn_client/estimated_odometry" />
    <param name="target0_topic" value="/ucl_1/vrpn_client/estimated_odometry" />
  </node>
</launch>
